backend/satellite_tools/reposync.py

```diff
@@ -360,14 +360,32 @@ def clear_ssl_cache():
     shutil.rmtree(ssldir, True)
 
 
+def verify_certificates_dates(certs):
+    cert = ""
+    isCert = False
+    valid = True
+    for line in cert_str.split('\n'):
```

Comment:
I think this should be:

```suggestion
    for line in certs.split('\n'):
```